daemon/src/projection.rs

```diff
@@ -261,11 +265,19 @@ impl Cfd {
                 (CfdState::ContractSetup, vec![])
             }
             ContractSetupCompleted { dlc } => {
-                self.details.tx_url_list.push(TxUrl::new(
-                    dlc.lock.0.txid(),
-                    network,
-                    TxLabel::Lock,
-                ));
+                // Lock transaction has shared output, but it likely has change
+                // output too that can be highlighted
+                let tx_url = TxUrl::new(dlc.lock.0.txid(), network, TxLabel::Lock);
+                // Find the relevant vout based on the locked amount as we don't
+                // have script pubkey for it.
+                let tx_url =
+                    if let Ok(outpoint) = dlc.lock.0.outpoint_of_value(dlc.lock_amount_for(role)) {
```

Comment:
I am afraid this is still not correct :sweat_smile: 

The output we are interested in is the _sum_ of both maker and taker. I'd suggest we expose a getter for that on `Dlc`.

Reply:
We can already access that, this is the other part of the tuple?